credsweeper/file_handler/data_content_provider.py

```diff
@@ -171,9 +173,107 @@ def _check_multiline_cell(self, cell: Tag) -> Optional[str]:
             # the cell will be analysed as multiline text
             self.line_numbers.extend(line_numbers)
             self.lines.extend(stripped_lines)
-            return " ".join(stripped_lines)
+            # additionally all line feeds in HTML transform to spaces
+            self.line_numbers.append(line_numbers[0])
+            self.lines.append(" ".join(stripped_lines))
```

Comment:
Could you explain more, why `self.lines` and `self.line_numbers` are appended here?
It seems those two variables already have been extended above.

Reply:
the duplication represent text as it is rendered 
https://github.com/Samsung/CredSweeper/blob/09137d8081f06c5e33b0ce3df120b65e1828fbb3/tests/samples/test.html#L82
In case of long line the join will be skipped, but previous cases are kept

Reply:
We communicated regarding this issue and decided to exclude this feature on this PR.